aiocache/decorators.py

```diff
@@ -310,19 +309,19 @@ async def decorator(
     ):
         missing_keys = []
         partial = {}
-        keys, new_args, args_index = self.get_cache_keys(f, args, kwargs)
+        orig_keys, cache_keys, new_args, args_index = self.get_cache_keys(f, args, kwargs)
 
         if cache_read:
-            values = await self.get_from_cache(*keys)
-            for key, value in zip(keys, values):
+            values = await self.get_from_cache(*cache_keys)
+            for orig_key, value in zip(orig_keys, values):
                 if value is None:
-                    missing_keys.append(key)
+                    missing_keys.append(orig_key)
                 else:
-                    partial[key] = value
+                    partial[orig_key] = value
```

Comment:
No, I think that `partial` should use the mapped keys returned by `key_builder`. These keys need to match the keys in the returned data, not the keys passed into the callable.

Reply:
> No, I think that `partial` should use the mapped keys returned by `key_builder`. These keys need to match the keys in the returned data, not the keys passed into the callable.

This is under the assumption of [Interpretation 2](https://github.com/aio-libs/aiocache/pull/564#issuecomment-1380574858).